Error: Cannot find module 'cli-engine/lib/commands'

Hi!

I tried using this package to update the readme for the heroku-slugs plugin, however I get the following error:
```
heroku-slugs $ heroku-plugin-readme-generator > README.md
module.js:487
    throw err;
    ^

Error: Cannot find module 'cli-engine/lib/commands'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Function.Setsumeisho.build (C:\Users\Ed\AppData\Roaming\npm\node_modules\heroku-plugin-readme-generator\src\setsumeisho.js:155:22)
    at Object.<anonymous> (C:\Users\Ed\AppData\Roaming\npm\node_modules\heroku-plugin-readme-generator\run.js:6:32)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
```

I'm presuming this package should declare a dependency on `cli-engine` in its package.json?

--- a/CODEOWNERS
+++ b/CODEOWNERS
@@ -1,0 +1,1 @@
+* @heroku/cli

--- a/package.json
+++ b/package.json
@@ -9,18 +9,8 @@
   "dependencies": {
     "@heroku/remark-heroku-bash-dollar": "^1.0.4",
     "@heroku/remark-heroku-command-headings-inline-code": "^1.0.5",
-    "lodash": "^4.17.4",
-    "remark-cli": "^4.0.0",
-    "remark-heroku-example-command": "^1.0.5",
-    "remark-heroku-sample-code": "^1.0.3",
-    "remark-heroku-topic-overview": "^1.0.3",
-    "remark-lint": "^6.0.0",
-    "remark-lint-are-links-valid": "^1.0.3",
-    "remark-preset-lint-recommended": "^3.0.0"
-  },
-  "devDependencies": {
-    "chai": "^3.5.0",
     "cli-engine": "3.5.58",
+    "cli-engine-heroku": "^6.0.1",
     "heroku-apps": "^2.4.2",
     "heroku-certs": "^1.1.40",
     "heroku-ci": "^1.8.2",
@@ -36,6 +26,17 @@
     "heroku-ps-exec": "^1.0.6",
     "heroku-run": "^3.4.12",
     "heroku-spaces": "^2.9.4",
+    "lodash": "^4.17.4",
+    "remark-cli": "^4.0.0",
+    "remark-heroku-example-command": "^1.0.5",
+    "remark-heroku-sample-code": "^1.0.3",
+    "remark-heroku-topic-overview": "^1.0.3",
+    "remark-lint": "^6.0.0",
+    "remark-lint-are-links-valid": "^1.0.3",
+    "remark-preset-lint-recommended": "^3.0.0"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
     "jest": "^20.0.4",
     "standard": "10.0.2"
   },
@@ -45,24 +46,6 @@
   ],
   "license": "ISC",
   "main": "run.js",
-  "peerDependencies": {
-    "cli-engine": "*",
-    "heroku-apps": "*",
-    "heroku-certs": "*",
-    "heroku-ci": "*",
-    "heroku-cli-addons": "*",
-    "heroku-cli-oauth": "*",
-    "heroku-cli-status": "*",
-    "heroku-fork": "*",
-    "heroku-git": "*",
-    "heroku-local": "*",
-    "heroku-orgs": "*",
-    "heroku-pg": "*",
-    "heroku-pipelines": "*",
-    "heroku-ps-exec": "*",
-    "heroku-run": "*",
-    "heroku-spaces": "*"
-  },
   "remarkConfig": {
     "plugins": [
       "remark-preset-lint-recommended",

--- a/src/setsumeisho.js
+++ b/src/setsumeisho.js
@@ -1,6 +1,7 @@
 // Setsumeisho literally means 'instructions' in Japanese
 
 const _ = require('lodash')
+const Flags = require('cli-engine-heroku').flags
 
 const packages = [
   'heroku-apps',
@@ -71,6 +72,16 @@ Setsumeisho.buildCommand = function (command) {
     lines.push('*' + desc + '*')
   }
   Setsumeisho.addAliases(lines, command)
+
+  // port needs/wants apps & orgs for V5 commands
+  if (command.needsApp || command.wantsApp) {
+    command.flags.app = Flags.app({required: !!command.needsApp})
+    command.flags.remote = Flags.remote()
+  }
+  if (command.needsOrg || command.wantsOrg) {
+    let opts = {required: !!command.needsOrg, hidden: false, description: 'organization to use'}
+    command.flags.org = Flags.org(opts)
+  }
 
   lines.push('')
   if (command.flags && command.flags.length) {

--- a/src/setsumeisho.js
+++ b/src/setsumeisho.js
@@ -75,12 +75,14 @@ Setsumeisho.buildCommand = function (command) {
 
   // port needs/wants apps & orgs for V5 commands
   if (command.needsApp || command.wantsApp) {
-    command.flags.app = Flags.app({required: !!command.needsApp})
-    command.flags.remote = Flags.remote()
+    if (!command.flags) command.flags = []
+    command.flags.push(Flags.app({required: !!command.needsApp}))
+    command.flags.push(Flags.remote())
   }
   if (command.needsOrg || command.wantsOrg) {
+    if (!command.flags) command.flags = []
     let opts = {required: !!command.needsOrg, hidden: false, description: 'organization to use'}
-    command.flags.org = Flags.org(opts)
+    command.flags.push(Flags.org(opts))
   }
 
   lines.push('')

